### DISCUSSION TITLE
Published before going live?

### DISCUSSION THREAD

**Discussion by Toderr** [mattwright324/youtube-metadata#137] [Q&A] [answered]
Hello, i want to check when did a live streaming start. The published date was 11 Mar 2023, but the actualStartTime was 13 Mar 2023. What is the difference about published date and actual start time?

The video ID is d6551xVX0UI

![image](https://user-images.githubusercontent.com/128206428/226106875-7b85887a-0ec7-4645-afbd-72deb0b6eeff.png)

![image](https://user-images.githubusercontent.com/128206428/226106887-6d9669ac-3611-443e-ba5d-5eaaa2d9f3d3.png)

**Comment by mattwright324** [comment, marked as answer]
Hi @Toderr 

What is happening is that the video was published/listed on March 11 but was scheduled for a future date of March 13 according to the `scheduledStartTime`. So in the time between the March 11 date and when it actually started, no video was viewable and it would have shown the scheduled date and time somewhere.

**Reply by Toderr** [reply]
So it was a pre recorded video not a live streaming?

**Reply by mattwright324** [reply]
In this case it looks like an actual livestream, not pre recorded. Not only is it really long but he is responding to comments.

Just looking at the metadata though it can be hard to tell if something was pre recorded after the fact / once the stream is over. If you were to look at the metadata before a scheduled stream was to go live and it had a Content Details `duration` then you could determine it was pre recorded.

**Reply by Toderr** [reply]
Oh okay then. Thank you very much. i really appreciate your help.
